ui/widgets/controlgroup.js

```diff
@@ -261,7 +266,11 @@ return $.widget( "ui.controlgroup", {
 				var instance = children[ value ]().data( "ui-controlgroup-data" );
 
 				if ( instance && that[ "_" + instance.widgetName + "Options" ] ) {
-					var options = that[ "_" + instance.widgetName + "Options" ]( value );
+					var options = that[ "_" + instance.widgetName + "Options" ](
+						children.length === 1 ?
+						"only" :
+						value
```

Comment:
These two lines should be indented, but I'd probably put the entire ternary on a single line.